### PR TITLE
feat(compute): custom vm type samples 3

### PR DESCRIPTION
**Pull request by FrodoTheTrue**

### CODE REVIEW

**Review by enocom — APPROVED** [review]

File: compute/instances/custom-machine-type/custom_machine_type_test.go [hunk 1]
```diff
@@ -166,4 +166,92 @@ func TestComputeCreateInstanceWithCustomMachineTypeSnippets(t *testing.T) {
 	if err != nil {
 		t.Errorf("deleteInstance got err: %v", err)
 	}
+
+	buf.Reset()
+
+	want = "Instance updated"
+
+	if err := createInstanceWithCustomMachineType(buf, tc.ProjectID, zone, instanceName, customMT); err != nil {
+		t.Errorf("createInstanceWithCustomMachineType got err: %v", err)
```
Comment (codyoss) [inline compute/instances/custom-machine-type/custom_machine_type_test.go:175]: for these test consider if the test is still valid if a precondition fails. For example if the instance is never created in this step you are unable to modify or get the instance so this should fail the test with a t.Fatalf.

**Comment by enocom** [comment]
@FrodoTheTrue There are some test errors that look pretty simple to fix. The sample itself looks good.

File: compute/instances/custom-machine-type/update_memory.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	"io"
+	"strings"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+// modifyInstanceWithExtendedMemory sends an instance creation request
+// to the Compute Engine API and wait for it to complete.
```
Comment (codyoss) [inline compute/instances/custom-machine-type/update_memory.go:30]: nit: waits

**Review by codyoss — CHANGES_REQUESTED** [review]
Just a couple small things, thanks for the PR.

**Review by codyoss — APPROVED** [review]

**Pull request merged**